### DISCUSSION TITLE
Bootstrap Icons Missing

### DISCUSSION THREAD

**Discussion by StevanFreeborn** [dotnet/docfx#10070] [General]
Anyone ever had an issue with bootstrap icons not showing up once the site is published?

The site seems to look fine when I serve it locally, but when I visit it on gh pages I see missing icons:

![image](https://github.com/dotnet/docfx/assets/65925598/d66ea7a4-9ac4-4ab6-9718-82d3dc775658)

I double checked that the static assets are all getting loaded properly.

Repo: https://github.com/StevanFreeborn/anthropic-client

**Comment by filzrev** [comment]
When opening site (https://anthropicclient.stevanfreeborn.com)
Following warning are displayed on Chrome Developer tools.

> Failed to decode downloaded font: <URL>
> anthropicclient.stevanfreeborn.com/:632 OTS parsing error: Failed to convert WOFF 2.0 font to SFNT
> anthropicclient.stevanfreeborn.com/:1 OTS parsing error: incorrect file size in WOFF header

It seems uploaded web font files(`.woff`/`.woff2`) are broken by git line feed conversions.
Because following files contents are different.
- https://anthropicclient.stevanfreeborn.com/public/bootstrap-icons-X6UQXWUS.woff2
- https://dotnet.github.io/docfx/public/bootstrap-icons-X6UQXWUS.woff2

I thought it can be fixed by explicitly specifying `.woff`/`.woff2` as binary at `.gitattributes` settings.

**Reply by StevanFreeborn** [reply]
Thank you so much for taking a look!

I got so focused on the network and sources tab I neglected to check the console. 🤦🏻‍♂️

Really appreciate it.
